tools/c7n_azure/c7n_azure/session.py

```diff
@@ -262,7 +272,7 @@ def load_auth_file(self, path):
                 client_id=data['credentials']['client_id'],
                 secret=data['credentials']['secret'],
                 tenant=self.tenant_id,
-                resource=self.resource_namespace
+                resource=self.base_url
```

Comment:
Where do we set `base_url`?

Reply:
Good catch, this was left over from initial draft